bindata/etcd/cluster-restore.sh

```diff
@@ -66,6 +66,18 @@ function wait_for_containers_to_stop() {
     echo "complete"
   done
 }
+function check_snapshot_status() {
+  dl_etcdctl
+  etcdctl snapshot status "${SNAPSHOT_FILE}" -w json > "${ETCD_DATA_DIR_BACKUP}"/etcd-backup-status-from-snapshot.json
+  tar -C "${ETCD_DATA_DIR_BACKUP}" -xzf "${BACKUP_FILE}" etcd-backup-status.json || true
+  # With the older backups, the backup status file may not exist.
+  if [ -f "${ETCD_DATA_DIR_BACKUP}/etcd-backup-status.json" ]; then
+    if ! diff -q "${ETCD_DATA_DIR_BACKUP}/etcd-backup-status.json" "${ETCD_DATA_DIR_BACKUP}/etcd-backup-status-from-snapshot.json" > /dev/null; then
```

Comment:
instead of writing the second file to verify could we ask etcdctl and pipe output as stdin to diff?

```
etcdctl snapshot status "${SNAPSHOT_FILE}" -w json | diff "${ETCD_DATA_DIR_BACKUP}/etcd-backup-status.json" - 
```